test/analysis/buildTypeWeights.test.ts

```diff
@@ -212,12 +211,18 @@ xdescribe('Test buildTypeWeightsFromSchema function', () => {
                     EMPIRE
                     JEDI
                 }`);
+
+            function reviews(multiplier: number): number {
+                return 1;
+            }
+
+            // TODO: Add tests for what the function does
             expect(buildTypeWeightsFromSchema(schema)).toEqual({
                 Query: {
                     weight: 1,
                     fields: {
                         // FIXME: check the best solution during implementation and update the tests here.
-                        reviews: (arg: number, type: Type) => arg * type.weight,
+                        reviews: expect.any(Function), // FIXME: Better way to test this?
```

Comment:
I'm ok with this. It allows the implementation to be more flexible with out expecting a very specific function definition. The functionality will be tested more specifically in the complexity analysis tests once properly made.